### ISSUE TITLE
Empty state reduced-motion is not working correctly

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#5530]
https://clayui.com/docs/components/empty-state.html#reduced-motion

reduced-motion = on
<img width="500" alt="image" src="https://github.com/liferay/clay/assets/7963804/300bbe54-1ea6-422b-9c87-1bed38e98158">

reduced-motion = off
<img width="500" alt="image" src="https://github.com/liferay/clay/assets/7963804/82043ee7-9600-420b-ae51-db2b1065a011">

Looks like is a CSS overwriting:

<img width="900" alt="image" src="https://github.com/liferay/clay/assets/7963804/061baf89-3c9d-4885-9f8b-06dc9908df3c">

<img width="900" alt="image" src="https://github.com/liferay/clay/assets/7963804/a0e7aab0-7167-4d56-a4ab-35227f31f88b">

The `@media (prefers-reduced-motion: reduce)` never overwrittes `.d-block-c-prefers-reduced-motion` and `.d-none-c-prefers-reduced-motion`

**Comment by marcoscv-work** [comment]
Related to: https://github.com/liferay/clay/issues/5477

**Comment by pat270** [comment]
@marcoscv-work in that example, the empty state image has the class `c-prefers-reduced-motion` on `c-empty-state`. This is Clay's manual override to force reduced motion even if the system setting prefers motion. The system setting shouldn't affect this. If you remove the class `c-prefers-reduced-motion`, it should display the animated image and when you toggle reduced motion it should show the unanimated image. Let me know if I'm not interpreting this correctly.

**Comment by marcoscv-work** [comment]
I think with the example and knowing how to apply it to portal will be perfect

**Comment by ethib137** [comment]
Hey guys, can we update the example to add the `c-prefers-reduced-motion` class to container, since it would normally be applied by adding this class to the body, not directly to specific elements? Also it looks like we are missing the https://clayui.com/images/success_state_reduced_motion.gif image. Can we fix that?

Additionally, is there any work that will be needed to apply this to DXP or will it only require releasing a new version of Clay?

**Comment by marcoscv-work** [comment]
The image in Liferay repo:
https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-theme/frontend-theme-unstyled/src/main/resources/META-INF/resources/_unstyled/images/states/success_state_reduced_motion.gif
